Jenkinsfile

```diff
@@ -9,9 +9,25 @@ pipeline {
             }
         }
 
+       stage("Yosys synthesis"){
+            steps {
+               sh 'docker run --rm -v $(pwd):/src -w /src librecores/librecores-ci:0.4.0 /bin/bash -c \
```

Comment:
Here you use 0.4.0, but in dockerrun you are not using a version.  Is that on purpose?

Reply:
For librecores-ci-openrisc, currently, there is no version! currently picks up latest